Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -166,22 +166,42 @@ public final class ProjectPamphletViewModel: ProjectPamphletViewModelType, Proje
       }
       .take(first: 1)
 
-    Signal.zip(
+    let freshProjectRefTagAndCookieRefTag: Signal<(Project, RefTag?, RefTag?), Never> = Signal.zip(
       freshProjectAndRefTag.skip(first: 1),
       self.viewDidAppearAnimated.signal.ignoreValues()
     )
     .map(unpack)
-    .map { project, refTag, _ in
-      let cookieRefTag = cookieRefTagFor(project: project) ?? refTag
+      .map { (arg) in
+        let (project, refTag, _) = arg
```

Comment:
Can we revert this argument destructuring? Usually when the compiler complains about that it's because there's a syntax error somewhere else.